reactnative-todo/src/components/common/mapComponentsToCallback.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { KBA } from './kba';
+import { Password } from './password';
+import { TermsAndConditions } from './specials';
+import { Username } from './username';
+import { TextField } from './text-field';
+
+function mapCallbacksToComponents(cb, idx) {
+  const name = cb?.payload?.input?.[0].name;
+  /** *********************************************************************
+   * SDK INTEGRATION POINT
```

Comment:
Let's call out which SDK is involved. Since there are technically two: the native bridge and the JavaScript.